ci_group/revolve2/ci_group/genotypes/cppnwin/modular_robot/v1/_body_develop.py

```diff
@@ -99,36 +87,38 @@ def __evaluate_cppn(
 
     # get rotation from output probabilities
     rotation_probs = [outputs[3], outputs[4]]
-    rotation = rotation_probs.index(min(rotation_probs))
+    rotation_index = rotation_probs.index(min(rotation_probs))
 
-    return (module_type, rotation)
+    return module_type, rotation_index
 
 
 def __add_child(
     body_net: multineat.NeuralNetwork,
     module: __Module,
-    child_index: int,
-    rotation: int,
-    grid: set[tuple[int, int, int]],
+    attachment_point_tuple: tuple[int, AttachmentPoint],
+    grid: NDArray[np.uint8],
 ) -> __Module | None:
-    forward = __rotate(module.forward, module.up, rotation)
-    position = __add(module.position, forward)
+    attachment_index, attachment_point = attachment_point_tuple
+
```

Comment:
we add children with respect to the candidate attachment point --> rotation depends on the attachment points rotation